course_info/tests/jasmine/controllers/DetailsControllerSpec.js

```diff
@@ -212,49 +214,337 @@ describe('Unit testing DetailsController', function () {
+        // doing the right thing, I left the code so when we
+        // revisit we can see what we tried to do. This will be
+        // included in the tech debt item TLT-2539
+        xit('should show regular non-editable text if called with ' +
             'editable=false or no editable attribute and show the model ' +
-            'value, not the form copy of the model value');
-        it('should show only show the label and the loading indicator ' +
+            'value, not the form copy of the model value', function(){
+            dc = $controller('DetailsController', {$scope: scope});
+            $httpBackend.expectGET(courseInstanceURL).respond(200, JSON.stringify({status: "success"}));
+            $httpBackend.expectGET(peopleURL).respond(200, JSON.stringify({status: "success"}));
+            $httpBackend.flush(2);
+            var liElement = directiveElem.find('li');
+            console.log(liElement);
+            expect(liElement.find('input')).not.toBeVisible();
+        });
+
+        // skipping the following test, they are tech debt in TLT-2539
+        xit('should only show the label and the loading indicator ' +
             'while loading, and hide the loading indicator when no ' +
-            'longer loading');
+            'longer loading', function(){
+        });
     });
 
     describe('fieldLabelWrapperDirective', function() {
-        it('should call backend with only the editable form fields');
-        it('should show a message if successful and update the local ' +
+        var compile, scope, directiveElem;
+        beforeEach(function () {
+             inject(function($compile, $rootScope){
+                compile = $compile;
+                scope = $rootScope.$new();
+              });
+              directiveElem = getCompiledElement();
+            scope = directiveElem.isolateScope();
+        });
+
+        function getCompiledElement(){
+            var element = angular.element('<hu-field-label-wrapper ' +
+                    'is-loading="dc.isUndefined(dc.courseInstance.term)"' +
+                    'field="test-name"' +
+                    'label="fieldLabelWrapperDirective test label">' +
+                '</hu-field-label-wrapper>');
+            var compiledElement = compile(element)(scope);
+            scope.$digest();
+            return compiledElement;
+        }
+
+        it('should display the correct value for label', function(){
+            dc = $controller('DetailsController', {$scope: scope});
+            $httpBackend.expectGET(courseInstanceURL).respond(200, JSON.stringify({status: "success"}));
+            $httpBackend.expectGET(peopleURL).respond(200, JSON.stringify({status: "success"}));
+            $httpBackend.flush(2);
+            var elem = directiveElem.find('li');
+            expect(elem.find('label').text().trim()).toBe('fieldLabelWrapperDirective test label');
+        });
+        // skipping the following tests, they are tech debt in TLT-2539
```

Comment:
I think this chunk of `if`s are the ones we agreed to remove completely -- they are effectively E2E (front end) testing that we'll accomplish with a combination of manual + selenium.